source/apis/routers .py

```diff
@@ -0,0 +1,50 @@
+from fastapi import HTTPException
+import models 
+from sqlalchemy.orm import Session
+from fastapi import Depends
+from database import get_db
+from schemas import Data_schema
+from fastapi import APIRouter
+
+router = APIRouter()
+
+@router.get("/")
+async def index(db: Session = Depends(get_db), skip: int = 0, limit: int = 100):
+    metadata = db.query(models.Metadata).offset(skip).limit(limit).all()
```

Comment:
All the controllers should only contain calls to services. 
You should create a services folder containing a metadata_service.py file.
In that file you write the logic.
However, all db queries should be implemented in separate functions in a queries file. The service functions then import and call those queries functions.